user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -87,23 +99,29 @@ The elements will be created with the following specifications:
+   >
+   > - The CSV file must be present on the server running the DMA.
+   > - The CSV file has to have the following structure: ElementName, Ccap_DMA, Collector_DMA, Protocol, IpAddress, GetCommunityString, SetCommunityString,GetCommunityStringColl, SetCommunityStringColl, Network, Market, Hub, SystemUser, SystemPass.
 
-1. Click *Run automation*.
+1. Click *Create Bulk*.
 
 The elements will be created with the specifications in the CSV file of each row.
 
 Here is an example of the CSV file with the mandatory headers and the content of each row.
 
-| ElementName | Be_DMA               | Collector_DMA        | Protocol                  | IpAddress    | CommunityString | Network | Market          | Hub           | SystemUser | SystemPass |
-|-------------|----------------------|----------------------|---------------------------|--------------|-----------------|---------|-----------------|---------------|------------|------------|
-| filename1   | EPM-BE-PUE-LAB-DMA03 | EPM-BE-PUE-LAB-DMA03 | CISCO CMTS CCAP Platform  | 127.0.0.100  | AnyText          | BAJIO   | AGUASCALIENTES | CTC-AGS       | us1        | Pass1      |
-| filename2   | EPM-BE-CLN-LAB-DMA04 | EPM-BE-PUE-LAB-DMA03 | Arris E6000 CCAP Platform | 127.0.0.101  | AnyText          | BAJIO   | AGUASCALIENTES | CTC-AGS       | us2        | Pass2      |
+| ElementName | Ccap_DMA    | Collector_DMA | Protocol                  | IpAddress    | GetCommunityString | SetCommunityString | GetCommunityStringColl | SetCommunityStringColl | Network         | Market          | Hub            | SystemUser | SystemPass |
+|-------------|-------------|----------------|---------------------------|--------------|---------------------|---------------------|------------------------|------------------------|-----------------|------------------|-----------------|-------------|------------|
+| filename1   | SLC-H62-G05 | SLC-H62-G05    | CISCO CBR-8 CCAP Platform | 127.0.0.100  | getPublic           | setprivate          | collectorget           | collectorset           | GLOBAL NETWORK  | EAST MARKET 01   | EAST HUB 01     | US1         | 123        |
+| filename2   | SLC-H62-G05 | SLC-H62-G05    | CISCO CBR-8 CCAP Platform | 127.0.0.101  | getprivate          | setPublic           | collectorget           | collectorset           | GLOBAL NETWORK  | EAST MARKET 01   | EAST HUB 01     | US2         | 123        |
 
 > [!NOTE]
-> These are the only values accepted for the **Protocol** column: CISCO CMTS CCAP Platform, Arris E6000 CCAP Platform, Casa Systems CCAP Platform, CISCO CBR-8 CCAP Platform, Huawei 5688-5800 CCAP Platform.
+>
+> - For the **Protocol** column, only the following values will be accepted: CISCO CMTS CCAP Platform, Arris E6000 CCAP Platform, Casa Systems CCAP Platform, CISCO CBR-8 CCAP Platform, Huawei 5688-5800 CCAP Platform.
+> - Click the *Cancel* button in any window to close the Automation script at any time.
```

Comment:
For the list of protocols, can we create a non-numeric list, as the one proposed above?